src/verify/hooks.ts

```diff
@@ -50,6 +52,8 @@ export function useVerifyPhoneNumber(phoneNumber: string, countryCallingCode: st
 
   const dispatch = useDispatch()
   const address = useSelector(walletAddressSelector)
+  const privateDataKey = useSelector(dataEncryptionKeySelector)
+  const publicDataKey = privateDataKey ? compressedPubKey(hexToBuffer(privateDataKey)) : null
```

Comment:
I'm making it required for this call on the backend.
At this point I think it's always present.

Reply:
does `null` count as a value, since it's not undefined? 😅 i think it will be always be available at this point but if for whatever reason it isn't, then null is still okay right? it doesn't affect too much functionality if the key is null, seems like it shouldn't be a blocker for going through phone verification?

Reply:
yeah i guess that makes sense. i'm not totally convinced that the DEK is important enough to block verification, but at the same time if the DEK hasn't been generated then that is probably a sign that there's something wrong with the account. i updated the hook to fail on the verification if there is no DEK